### DISCUSSION TITLE
/users/me returns 403 forbidden

### DISCUSSION THREAD

**Discussion by klaydev** [fastapi-users/fastapi-users#1310] [Q&A] [answered]
Hi. When I try to get a user by /users/me (login is done), I get a 403 code, while I wrote my get handler, which also requires authorization and returns the user and it works. Here it is:
```py
@app.get('/test')
async def test_route(user: User = Depends(current_active_user)):
    return {"user": user}
```
I did everything as shown on the wiki

**Comment by frankie567** [comment, marked as answer]
Is it possible that you set `requires_verification=True` on your users router? https://fastapi-users.github.io/fastapi-users/12.1/configuration/routers/users/#optional-user-verification

If so, the `/me` route will raise a 403 until the email is verified for the logged in user.
